apis/org_handler_test.go

```diff
@@ -0,0 +1,154 @@
+package apis_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"code.cloudfoundry.org/cf-k8s-api/apis"
+	"code.cloudfoundry.org/cf-k8s-api/apis/fake"
+	"code.cloudfoundry.org/cf-k8s-api/repositories"
+	. "github.com/onsi/gomega"
+	"github.com/sclevine/spec"
+)
+
+var _ = SuiteDescribe("listing orgs", func(t *testing.T, when spec.G, it spec.S) {
+	const (
+		rootURL = "https://api.example.org"
+	)
+
+	var (
+		ctx       context.Context
+		orgHander *apis.OrgHandler
+		orgRepo   *fake.CFOrgRepository
+		req       *http.Request
+		rr        *httptest.ResponseRecorder
+		err       error
+	)
+
+	it.Before(func() {
+		ctx = context.Background()
+		orgRepo = new(fake.CFOrgRepository)
+		orgHander = apis.NewOrgHandler(orgRepo, rootURL)
+		rr = httptest.NewRecorder()
+
+		req, err = http.NewRequestWithContext(ctx, http.MethodGet, "", nil)
+		g.Expect(err).NotTo(HaveOccurred())
+	})
+
+	it("sets the json content type", func() {
+		http.HandlerFunc(orgHander.OrgListHandler).ServeHTTP(rr, req)
```

Comment:
In other handler tests we're now calling `handler.RegisterRoute(router); router.ServeHTTP(rr, req)` instead. This exercises the route and ensures that it's wired up correctly. It also lets us use parameters that might be passed in the path, like GUID

Reply:
Yep - that's good. I missed that when merging the routes change.